src/decorators/publicL2OpStackActions.ts

```diff
@@ -20,6 +22,23 @@ export type PublicL2OpStackActions<
   getProveWithdrawalTransactionArgs: (
     args: getProveWithdrawalTransactionArgsParams,
   ) => Promise<getProveWithdrawalTransactionArgsReturnType>
+  /**
+   * Estimate the l1 gas price portion for a transaction
+   * @example
+   * const price = await getL1GasPrice(publicClient, {
+   *   blockNumber,
+   *   blockTag,
+   * });
+   */
```

Comment:
```suggestion
```
Maybe delete for now and go through and add jsdocs in it's own pr

Reply:
yeah I've been underinvesting in jsdoc in favor of the markdown docs, not sure if right tradeoff.

Reply:
Jsdoc is more important because you see jsdoc in your editor imo